File: variablesCsharp/7.cs
using System;


namespace Variables
{
    internal class Siete
    {
        public Siete() {
            //área del cuadrado
            Console.WriteLine("Digite la longitud del lado del cuadrado: ");
            int longitud = int.Parse(Console.ReadLine());

            int area = longitud * longitud;

            Console.WriteLine("El área del cuadrado es igual a: " +  area);
        }
    }
}

File: dart/variables/7.dart
import 'dart:io';

void main() {
  print("Calcular el área de un cuadrado");

  print("Digite la longitud del cuadrado: ");

  int long = int.parse(stdin.readLineSync()!);

  int area = long * long;

  print("El área del cuadrado con longitud ${long} es igual a ${area}");
}
